timemachine/fe/system.py

```diff
@@ -54,6 +92,7 @@ def get_U_fn(self):
 
         def U_fn(x):
             Us_vdw, Us_coulomb = nbpl_U(x)
+            # (ytz): revert!
```

Comment:
revert!

Reply:
lol - this was a spurious comment! willfix